install

```diff
@@ -49,10 +49,16 @@ do
   esac
 done
 
+source .env
 PLACE_DOMAIN=${PLACE_DOMAIN:="localhost:8080"}
 PLACE_EMAIL=${PLACE_EMAIL:="support@place.tech"}
 PLACE_PASSWORD=${PLACE_PASSWORD:="development"}
 
+if [ ! -f ./ssl/default-domain/fullchain.pem ]; then
+  echo "=== Generating SSL certificates..."
+  ./scripts/generate-ssl ${PLACE_DOMAIN} 2> /dev/null || (printf "\nfailed to generate SSL certificates\n" && exit 1)
+fi
+
```

Comment:
Since this is idempotent, may as well throw it into the `restart` script as well.
Ideally we should only have one script.